eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -118,3 +121,11 @@ services:
             - [setFragmentPath, [%fragment.path%]]
             - [setSiteAccess, [@ezpublish.siteaccess]]
         abstract: true
+
+    ezpublish.converter.repository:
```

Comment:
Yes please, `ezpublish.param_converter`. Examples are good for inspiration, but don't forget that we have **hundreds** of services...